doc/source/project/issues.rst

```diff
@@ -122,10 +138,12 @@ particular domain, as well as more organized release notes.
 - ``area/histories``
 - ``area/i18n`` - Internationalization and localization
 - ``area/jobs``
+- ``area/jobs/kubernetes``
 - ``area/libraries`` - Change related to data libraries
 - ``area/objectstore``
 - ``area/performance``
-- ``area/reports``
+- ``area/reports`` - The reports webapp
+- ``area/rules`` - Rule builder
```

Comment:
Maybe this could be a subtopic of `area/UI-UX` ?

Reply:
I considered this and would have done it, I just didn't know if area/UI-UX made more sense or area/client as a base. I generally use area/UI-UX but I don't know that it is a consensus. I'm happy for more structure and more descriptions if someone wants to follow up.

Reply:
I'm not sure what's the difference between `area/client` and `area/UI-UX`, actually `area/client` is not even listed here.